mogc_partnerships/permissions.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.permissions import BasePermission
+
+from .lib import get_cohort
+
+class ManagerCreatePermission(BasePermission):
+
+    managed_methods = ("POST")
+
+    def has_permission(self, request, view):
+        if not request.method in self.managed_methods:
+            return True
+
+        user = request.user
+        serializer = view.get_serializer(data=request.data)
+        if not serializer.is_valid():
+            return False
+
+        partner = serializer.validated_data["partner"]
```

Comment:
🤔 seems like we should probably have the partner ID in the API routes

Reply:
If this is the best we can get at the moment than that is good enough for me. I'd encourage creating an issue from your comment (using _reference in a new issue_) so that we remember it during the next clean-up opportunity. As long as appropriate tests are present you (or another developer) can refactor fearlessly at any future time.